Bottom: ed07774027279e8dcef4ee675b18f0fe391e53c3
Top:    4895cf70cf031fbcb4acc943a773ac3949293ae4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-28 23:32:28 +0100

Refresh of hid-png-don-t-use-the-core-s-d

---

diff --git a/src/hid/png/png.c b/src/hid/png/png.c
index d671a7e..13b76a0 100644
--- a/src/hid/png/png.c
+++ b/src/hid/png/png.c
@@ -447,7 +447,7 @@ set_layer (const char *name, int group)
 }
 
 static void
-png_expose (const BoxType *bounds)
+png_expose (BoxType *bounds)
 {
   HID *old_gui = gui;
   int group;
